package.json

```diff
@@ -65,7 +70,7 @@
     "node-fetch": "^2.6.7"
   },
   "devDependencies": {
-    "check-dependency-version-consistency": "2.0.0",
+    "check-dependency-version-consistency": "3.0.0",
```

Comment:
Needed to support package.json → workspaces → nohoist
- https://github.com/bmish/check-dependency-version-consistency/pull/381 ()